ansible/roles/packages/tasks/redhat.yaml

```diff
@@ -65,9 +65,44 @@
     - versionlock_plugin_enabled
     - item in exportedversionlocklist.stdout
 
+- name: check kubernetes rpms exist for provided version
+  stat:
+    path: "/opt/{{ 'kubectl-' + package_versions.kubernetes_rpm }}{{ '-fips' if fips.enabled else '' }}.rpm"
+  delegate_to: localhost
```

Comment:
nit: its good to check all the kubernetes rpms that are required for this task.

Reply:
done